Sources/StreamChat/Query/Filter.swift

```diff
@@ -139,14 +143,16 @@ public struct Filter<Scope: FilterScope> {
     public init(
         operator: String,
         key: String?,
-        value: FilterValue
+        value: FilterValue,
+        isACollectionFilter: Bool
```

Comment:
This is probably enough:
```suggestion
        isCollectionFilter: Bool
```